core/types/state_account_storage.libevm_test.go

```diff
@@ -107,7 +116,10 @@ func TestStateAccountExtraViaTrieStorage(t *testing.T) {
 		{
 			name: "arbitrary payload",
 			registerAndSetExtra: func(a *types.StateAccount) (*types.StateAccount, assertion) {
-				e := types.RegisterExtras[types.NOOPHeaderHooks, *types.NOOPHeaderHooks, arbitraryPayload]()
+				e := types.RegisterExtras[
+					types.NOOPHeaderHooks, *types.NOOPHeaderHooks,
+					types.NOOPBodyHooks, *types.NOOPBodyHooks,
+					arbitraryPayload]()
```

Comment:
```suggestion
					arbitraryPayload,
				]()
```

Reply:
This makes me think, given there is now gofumpt (built-in gopls and has a golangci-lint linter), should we use that for `*libevm*` files? It's a bit stricter than gofmt, it might help for those formatting issues 🤔